### ISSUE TITLE
关于应用在别的网络结构

### ISSUE THREAD

**Issue by MercuialC** [PRIS-CV/Mutual-Channel-Loss#13]
感谢您优秀的工作，若应用于resnet50上，第252行的class model_bn(nn.Module):需要作何修改呢？

**Comment by dongliangchang** [comment]
你是指从头训练的情况么? 这个地方不需要修改，只需将resnet50最终输出的特征channel数改为600即可，参考169行。

**Comment by MercuialC** [comment]
感谢您的回复！
我尝试在resnet结构中加入了
`def resnet50(pretrained=False, **kwargs):
    """Constructs a ResNet-50 model.

    Args:
        pretrained (bool): If True, returns a model pre-trained on ImageNet
    """
    model = ResNet(Bottleneck, [3, 4, 6, 3], **kwargs)
    if pretrained:
        model.load_state_dict(model_zoo.load_url(model_urls['resnet50']))
    return model`

由于瓶颈块expansion = 4，导致最后张量维度为2400与600不匹配，此时应如何修改具体输出特征channel数呢？

**Comment by dongliangchang** [comment]
改成150就行了吧，150*4=600。最终输出的特征channel数为600即可。MC-Loss约束的是瓶颈特征。

**Comment by MercuialC** [comment]
layer1-layer3不需要修改，会造成影响吗

**Comment by dongliangchang** [comment]
> 
> 
> layer1-layer3不需要修改，会造成影响吗

应该没有
